app/controllers/individual/IndNonUKAddressWithoutIdController.scala

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.individual
+
+import controllers.actions._
+import forms.AddressWithoutIdFormProvider
+import models.{Country, Mode}
+import navigation.Navigator
+import pages.IndAddressWithoutIdPage
+import play.api.Logging
+import play.api.i18n.{I18nSupport, MessagesApi}
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import repositories.SessionRepository
+import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendBaseController
+import utils.CountryListFactory
+import views.html.individual.IndAddressWithoutIdView
+
+import javax.inject.Inject
+import scala.concurrent.{ExecutionContext, Future}
+
+class IndNonUKAddressWithoutIdController @Inject() (
+  override val messagesApi: MessagesApi,
+  countryListFactory: CountryListFactory,
+  sessionRepository: SessionRepository,
+  navigator: Navigator,
+  standardActionSets: StandardActionSets,
+  formProvider: AddressWithoutIdFormProvider,
+  val controllerComponents: MessagesControllerComponents,
+  view: IndAddressWithoutIdView
+)(implicit ec: ExecutionContext)
+    extends FrontendBaseController
+    with I18nSupport
+    with Logging {
+
+  val countriesList: Option[Seq[Country]] = countryListFactory.countryListWithoutUKCountries
+
+  def onPageLoad(mode: Mode): Action[AnyContent] = standardActionSets.identifiedUserWithData() {
+    implicit request =>
+      countriesList match {
+        case Some(countries) =>
+          val form = formProvider(countries)
+          val preparedForm = request.userAnswers.get(IndAddressWithoutIdPage) match {
+            case None        => form
+            case Some(value) => form.fill(value)
+          }
+
+          Ok(
+            view(
+              preparedForm,
+              countryListFactory.countrySelectList(form.data, countries),
+              routes.IndNonUKAddressWithoutIdController.onSubmit(mode),
```

Comment:
There's no need to pass the Call to the view, you can just use it directly in the view